app/actions/remote/session.ts

```diff
@@ -245,7 +245,7 @@ export const sendPasswordResetEmail = async (serverUrl: string, email: string) =
         return {error};
     }
     return {
-        data: response.data,
+        data: response.status,
```

Comment:
If what we are returning and checking is the status, I would also change the name of the variable, as:
```suggestion
        status: response.status,
```